phoenix-protocol/src/main/MetaDataService.proto

```diff
@@ -144,11 +141,13 @@ message DropSchemaRequest {
 message AddColumnRequest {
   repeated bytes tableMetadataMutations = 1;
   optional int32 clientVersion = 2;
+  optional PTable parentTable = 3;
```

Comment:
Need to send over the parent PTable while dropping or adding a column to a child view (so that we can construct the complete metadata on the server side)